ci-operator/config/openshift/sriov-dp-admission-controller/openshift-sriov-dp-admission-controller-master__okd-scos.yaml

```
@@ -0,0 +1,39 @@
+base_images:
+  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+  origin_scos-4.15_base-stream9:
+    name: scos-4.15
+    namespace: origin
+    tag: base-stream9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.rhel7
+  from: origin_scos-4.15_base-stream9
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+  to: sriov-dp-admission-controller
+promotion:
+  to:
+  - name: sriov-4.16
+    namespace: origin
+releases:
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.16"
```

Let's make sure we always test against the SCOS payload IS
```suggestion
      name: "scos-4.16"
```